plugins/com.python.pydev.docs/homepage/final/developers.html

```diff
@@ -304,7 +304,7 @@ <h2>Repository</h2>
 above (i.e. clone a project, create a branch, edit and send pull a request):</p>
 <ul class="simple">
 <li><a class="reference external" href="https://codeburst.io/a-step-by-step-guide-to-making-your-first-github-contribution-5302260a2940">Make a contribution using github</a></li>
-<li><a class="reference external" href="https://github.com/Roshanjossey/first-contributions/blob/master/github-desktop-tutorial.md">Make a contribution using github (using github desktop)</a></li>
+<li><a class="reference external" href="https://github.com/firstcontributions/first-contributions/blob/master/gui-tool-tutorials/github-desktop-tutorial.md#first-contributions">Make a contribution using github (using github desktop)</a></li>
```

Comment:
Thanks for the fix, still, this file is actually generated... the proper place to fix this is in the `.rst` (i.e.: https://github.com/fabioz/Pydev/blob/master/plugins/com.python.pydev.docs/homepage/developers.contents.rst)

Reply:
Should I remove the changes to this file from the PR?

Reply:
You can keep it (it doesn't make much difference as it'll be overwritten when the homepage is actually updated).